app/bundles/LeadBundle/Controller/LeadController.php

```diff
@@ -1381,11 +1382,18 @@ public function emailAction($objectId = 0)
 
                     $bodyCheck = trim(strip_tags($email['body']));
                     if (!empty($bodyCheck)) {
+                        /** @var MailHelper $mailer */
                         $mailer = $this->get('mautic.helper.mailer')->getMailer();
 
                         // To lead
                         $mailer->addTo($leadEmail, $leadName);
 
+                        if (!empty($email[EmailType::REPLY_TO_ADDRESS])) {
```

Comment:
It would be nice to have this if covered with tests

Reply:
Am I able to do it with functional tests somehow?

Reply:
It will be quite simple once we merge https://github.com/mautic/mautic/pull/11613

See https://symfony.com/doc/5.4/mailer.html#write-a-functional-test